ScreenSleeper/Shared/Application.cs

```diff
@@ -5,9 +5,13 @@ namespace Shared
 {
     public static class Application
     {
-        private static readonly Mutex _mutex = new(true, name: "{04CF3278-CC61-4716-8B67-955C0FD7C469}");
```

Comment:
I think you were right before; the `Mutex` needs to be a member so that it is not disposed. (If it's disposed, another instance will be able to create it again.)
```
private static readonly Mutex _mutex = new(true, name: "{04CF3278-CC61-4716-8B67-955C0FD7C469}", createdNew: out _isFirst);
```